repos/c2sm/packages/fdb-fortran/package.py

```diff
@@ -11,10 +11,7 @@ class FdbFortran(CMakePackage):
     version('0.1.0', tag='0.1.0')
 
     depends_on('cmake@3.10:', type='build')
-    depends_on('eckit')
-    depends_on('metkit')
-    depends_on('eccodes +fortran')
-    depends_on('fdb@5.11.0:')
+    depends_on('fdb@5.11.0: ^eccodes +fortran')
```

Comment:
I don't think this is a wise change. fdb-fortran directly depends on fdb5, eckit, metkit (https://github.com/MeteoSwiss/fdb-fortran/blob/main/CMakeLists.txt#L17-L19). So by directly listing them as dependencies in spack, the requirements of all listed versions of fdb-fortran will get their required dependencies.
The change would make this package description depend on an implementation detail of fdb, namely that it depends on fdb, eckit, metkit. But fdb@27 may drop one of the dependencies, thus the statement `depends_on fdb@5.11.0:` is not sufficient to satisfy fdb-fortran's requirements, as we do not know what future versions of fdb will bring.